Pod/Sources/OriginateMutableRemoteCollection.m

```diff
@@ -0,0 +1,162 @@
+//
+//  OriginateMutableRemoteCollection.m
+//  OriginateRemoteCollection-Example
+//
+//  Created by Philip Kluz on 10/03/15.
+//  Copyright © 2015 originate.com. All rights reserved.
+//
+
+#import "OriginateMutableRemoteCollection.h"
+#import "OriginateRemoteCollection+Internal.h"
+
+@interface OriginateMutableRemoteCollection ()
+{
+    NSMutableArray *_objects;
+}
+
+@property (nonatomic, strong, readwrite) NSMutableArray *objects;
+
+@end
+
+@implementation OriginateMutableRemoteCollection
+
+@dynamic objects;
+@dynamic delegate;
+
+
+#pragma mark - OriginateRemoteCollection
+
+- (instancetype)init
+{
+    self = [super init];
+    
+    if (self) {
+        _objects = [NSMutableArray array];
+    }
+    
+    return self;
+}
+
+- (NSMutableArray *)objects
+{
+    return _objects;
+}
+
+- (void)setObjects:(NSMutableArray *)objects
+{
+    [super setObjects:objects];
+}
+
+- (NSMutableArray *)allObjects
+{
+    return self.objects;
```

Comment:
They are/were. `allObjects` in fact is not necessary. `setObjects` is not currently being used by is the getter of the overridden property is not specialized as here, it returns the `NSArray` of the superclass